content/spin/v2/cli-reference.md

```diff
@@ -4651,6 +5367,87 @@ HTTP TRIGGER OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v2.7"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin up --help
+
+spin-up 
+Start the Spin application
+
+USAGE:
+    spin up [OPTIONS]
+
+OPTIONS:
+        --build                    For local apps, specifies to perform `spin build` before running
+                                   the application [env: SPIN_ALWAYS_BUILD=]
+        --cache-dir <CACHE_DIR>    Cache directory for downloaded components and assets
+        --direct-mounts            For local apps with directory mounts and no excluded files, mount
+                                   them directly instead of using a temporary directory
+    -e, --env <ENV>                Pass an environment variable (key=value) to all components of the
+                                   application
+    -f, --from <APPLICATION>       The application to run. This may be a manifest (spin.toml) file,
+                                   a directory containing a spin.toml file, or a remote registry
+                                   reference. If omitted, it defaults to "spin.toml"
+    -h, --help                     
+    -k, --insecure                 Ignore server certificate errors from a registry
+        --temp <TMP>               Temporary directory for the static assets of the components
+
+HTTP TRIGGER OPTIONS:
```

Comment:
I am confused by this because these are common trigger options not HTTP trigger options.  We have a separate section for HTTP trigger options.  (See line 5456.)  But this seems to be existing so I guess out of scope for this PR.